FFXIVClientStructs/FFXIV/Client/Game/InstanceContent/InstanceContentDeepDungeon.cs

```diff
@@ -25,6 +26,17 @@ public unsafe partial struct InstanceContentDeepDungeon {
     [FieldOffset(0x1F0A)] public byte SyncedGearLevel;
     [FieldOffset(0x1F0B)] public byte HoardCount;
 
+    [FieldOffset(0x28CE)] public byte DeepDungeonId; // 1-3
+
+    [FieldOffset(0x2900), FixedSizeArray] internal FixedSizeArray25<byte> _mapData; // list of RoomFlags
```

Comment:
```suggestion
    [FieldOffset(0x2900), FixedSizeArray] internal FixedSizeArray25<RoomFlags> _mapData;
```
Use the type it is instead of creating it without usage. 

If this breaks the Generator @aers is responsible for not having implemented checking the size for enums.